File: python/1048.py
# -*- coding: utf-8 -*-

n = float(input())

if (0 < n <= 400):
    s = n*15/100+n
    p = 15
elif (400 < n <= 800):
    s = n*12/100+n
    p = 12
elif (800 < n <= 1200):
    s = n*10/100+n
    p = 10
elif (1200 < n <= 2000):
    s = n*7/100+n
    p = 7
else:
    s = n*4/100+n
    p = 4
r = n*p/100

print('''Novo salario: %0.2f
Reajuste ganho: %0.2f
Em percentual: %r'''%(s, r, p) + ' %')

File: categorias/iniciante/c++/1048.cpp
#include <iostream>

using namespace std;

int main() {
    double salario, novoSalario, ajuste;
    int per;
    cout.precision(2);
    cout.setf(ios::fixed);

    cin >> salario;
    if (salario <= 400) {
        per = 15;
        ajuste = (per/100.0) * salario;
        novoSalario = salario + ajuste;
    } else if (salario <= 800) {
        per = 12;
        ajuste = (per/100.0) * salario;
        novoSalario = salario + ajuste;
    } else if (salario <= 1200) {
        per = 10;
        ajuste = (per/100.0) * salario;
        novoSalario = salario + ajuste;
    } else if (salario <= 2000) {
        per = 7;
        ajuste = (per/100.0) * salario;
        novoSalario = salario + ajuste;
    } else if (salario > 2000) {
        per = 4;
        ajuste = (per/100.0) * salario;
        novoSalario = salario + ajuste;
    }

    cout << "Novo salario: " << novoSalario << "\n";
    cout << "Reajuste ganho: " << ajuste << "\n";
    cout << "Em percentual: " << per << " %\n";
    return 0;
}